forge-gui/res/cardsfolder/upcoming/geths_summons.txt

```diff
@@ -0,0 +1,11 @@
+Name:Geth's Summons
+ManaCost:2 B B
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TargetMin$0 | TargetMax$ 1 | TgtPrompt$ Choose up to one creature card in your graveyard | SubAbility$ DBReturnCorrupted | ValidTgts$ Creature.YouCtrl | SpellDescription$ Return up to one target creature card from your graveyard to the battlefield. Corrupted — For each opponent who has three or more poison counters as you cast this spell, put up to one target creature card from that player’s graveyard onto the battlefield under your control.
+SVar:DBReturnCorrupted:DB$ ChangeZone | Origin$ Graveyard | TargetMin$ 0 | TargetMax$ OneEach | TgtZone$ Graveyard | GainControl$ True | Destination$ Battlefield | TgtPrompt$ Choose target card in an opponent's graveyard | ValidTgts$ Creature.RememberedPlayerCtrl | TargetsWithDifferentControllers$ True
+T:Mode$ SpellCast | ValidCard$ Card.Self | Static$ True | Execute$ CountCorruptedPlayers
+SVar:CountCorruptedPlayers:DB$ Pump | RememberObjects$ Opponent.IsCorrupted
+SVar:OneEach:Count$RememberedSize
+DeckNeeds:Ability$Graveyard
```

Comment:
I feel like such DeckNeeds is way too strict

A creature dying is normal gameplay